AK/LexicalPath.cpp

```diff
@@ -82,7 +82,7 @@ String LexicalPath::canonicalized_path(String path)
         return ".";
 
     // NOTE: If there are no dots, no '//' and the path doesn't end with a slash, it is already canonical.
-    if (!path.contains("."sv) && !path.contains("//"sv) && !path.ends_with('/'))
+    if (!path.contains("."sv) && !path.contains("//"sv) && !path.ends_with('/')) [[likely]]
```

Comment:
re `[[likely]]`, let's not micro-optimise here, if we hit a case where this is a bottleneck, we'll add the annotation, or rework the method somehow.

Reply:
Yeah, you're right, this isn't in a loop.